Salv mining tweaks

RorororoIDK opened the pull request:
many changes to hardsuits relating to salv and mining, as well as ticket vendor changes and other tweaks.

<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
Changes to most of the Hardsuits in Salvage and Mining, as well as ticket vendor tweaks and rebalances.
Removes bubble shield from Maxim because it was bugged to hell and wildly overpowered (it literally made ai ignore you and gave you free jets)
Remove Jhawk and its ammo from ticket vendor, replaced with a nerfed Stechkin and its ammo which now also starts with .40 sp so birds arent locked out of a sidearm option.
Added advanced mineral scanner to mining ticket vendor.
Lowered Cycler shotgun max ammo count so its more in line with combat shotty and not a Bulldog-lite.
Raised the price of and lowered values of the Merc web vest.
Changed immolator description since it was just base laser carbine description.

## Why we need to add this
Salvage, quite literally, had a better bloodred hardsuit, I am not overexaggerating this, the maxim was a buggy, overpowered mess, and it needed fixed ages ago, with this change it still remains a combat suit but not literally better than bloodred, caps, secs and literally every other hardsuit outside commander hardsuit, removed its bubble shield because it made no sense for salv to have this, at all, yes im sure it was cool, but this thing was genuinely insane, it gave you free jets in space and made ai ignore you, which is silly considering all salv fights is ai, this isnt even counting the fact its a literal bubble shield and protected you even more on top of the already insane stats for the maxim.

While lowering the values for salvager hardsuits where it made sense, i raised the values for mining suits where it mattered, i raised slash and blunt for mining suits and left their pierce low since they dont really need pierce resist and id rather not have salv pt.2.

Removed Jhawk in the ticket vendor because why the hell did salv have it, replaced it with a nerfed stechkin (worse than Due process now) that was formerly sec contra, yes im aware its in uplink but theres better option in there too if you want a silenced pistol, changed it to SalvSec contra and changed its starting ammo from fmj to sp.

Lowered Cycler shotgun max ammo count from 9 to 7 so its in line with combat shotty, this thing was pretty strong when paired with speedloaders, it was basically just a bulldog-lite, its still very good but this just neatens up the values a bit.

Raised price of and slightly lowered the stats for Merc web vest, it was 200 tickets in the vendor and was literally just better than sec armor, its still good for exped since it has no slowdown while maxim does now, also lowered its stam resist massively, it only tanks one more stun baton or disabler shot now.

## Media (Video/Screenshots)
<img width="406" height="364" alt="Screenshot 2025-11-13 061058" src="https://github.com/user-attachments/assets/a8d96a7b-bf4e-445d-9ac7-f8285cff5471" />


## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
:cl: Roro
- remove: Removed maxim bubble shield
- tweak: Removed Jhawk and its ammo from Salv ticket vendor, changed its contra label back to syndicate
- tweak: Tweaked resist for maxim, 40% blunt to 35%, 40% slash to 35%, 50% pierce to 35%, 70% heat to 40%, 90% rad to 70%, 80% explosive resist to 60%, also added 20% caustic since it had none
- tweak: Gave Maxim some actual slowdown as well as some slowdown for holding it.
- tweak: Tweaked resist for Salvager hardsuit, 30% blunt to 20%, 40% slash to 20%, 35% pierce to 25%
- tweak: Tweaked resists for Mining hardsuit, 15% blunt to 25%, 15% slash to 25%
- tweak: Tweaked resists and speed for Advanced Mining hardsuit, 25% blunt to 35%, 25% slash to 35%, 10% slowdown to 15%, also added slowdown when holding it in hand
- tweak: Tweaked Merc web vest resists and price, from 200 tickets to 350, 50% pierce to 40%, 45% stam resist to 15%
- tweak: Lowered ammo for cycler shotgun from 9 to 7
- tweak: Changed Immolators description so its not the same as Laser Carbines
- tweak: Space Hazard suit can now be used to craft a goliath hardsuit
- tweak: Added MK58 and its ammo to salv ticket vendor to replace the Jhawk, MK58 now has SalvSec contra label

RorororoIDK committed:
Rebalances to Salv and Mining

many changes to hardsuits relating to salv and mining, as well as ticket vendor changes and other tweaks.

RorororoIDK committed:
forgot the labels oopsies

jus adding the #Starlight beside some hardsuit changes cus theyre not in _starlight

mikeysaurus commented:
Few things I thought of:

- Ideally, make a dupe of the merc vest, rename it. Leave the original merc items as is for admemes. Peter on the discord is working on new sprites for salvage themed gear that will replace the merc gear in the store, it just isnt done yet. I think the balance changes make sense, I'd just dupe the item for now to not affect the merc gear. (This was a request to me too by the event staff)
- I do not think you should be able to purchase the adv scanner at all. We should encourage cross-department activity like requesting gear from science. The default scanner is fine for purchase but advanced should not be available IMO
- Can the Bubble shield be adjusted to fix the AI behavior instead of removing? I think it's pretty unique, would be a shame to outright remove.
- I purposely kept the mining suits lower resistances to avoid encouraging combat, and to avoid giving antags potential for strong armor.
- Space Hazard -> Goliath suit should have its own variant of a mining suit. The purpose of the seperate suits is to maintain seperate department identities especially if Mining can be antags.

RorororoIDK commented:
> Few things I thought of:
> 
> * Ideally, make a dupe of the merc vest, rename it. Leave the original merc items as is for admemes. Peter on the discord is working on new sprites for salvage themed gear that will replace the merc gear in the store, it just isnt done yet. I think the balance changes make sense, I'd just dupe the item for now to not affect the merc gear. (This was a request to me too by the event staff)
> * I do not think you should be able to purchase the adv scanner at all. We should encourage cross-department activity like requesting gear from science. The default scanner is fine for purchase but advanced should not be available IMO
> * Can the Bubble shield be adjusted to fix the AI behavior instead of removing? I think it's pretty unique, would be a shame to outright remove.
> * I purposely kept the mining suits lower resistances to avoid encouraging combat, and to avoid giving antags potential for strong armor.
> * Space Hazard -> Goliath suit should have its own variant of a mining suit. The purpose of the seperate suits is to maintain seperate department identities especially if Mining can be antags.

- The maxim bubble shield, even without its bugs is crazy strong, salv has easy access to higher cap batteries so they can tank massive amounts of damage with just the shield alone, I agree that its a cool feature but it is just so strong and they never should have been given it, even without it, this nerfed maxim is still on par with Caps hardsuit.
- I agree completely on the space hazard but theres 2 issues on it, firstly, the space hazard itself needs new sprites, it has no light, it has no sprites for seperate species, it is *so* old, ideally the space hazard gets new sprites first before the goliath, secondly, I am not a spriter, I can look into learning it but that would take time and I'd rather just add this now for the time being as its just something cool for Mining to get, even if strong, goliath suit isnt insane, while yes, it has 50% pierce, the rest of its stats are nothing crazy considering the effort that goes into acquiring it, also, Mining antags can literally just take the spatio, some Miners already do this for the light alone when mining, if they wanted the goliath suit they can already get it, this just makes it more fair for them
- As for the Advanced mineral scanner, I added it because, from my own experience and the experience of other Mining players, science is never guaranteed, and frankly, mining does not have that much in their ticket vendor so this is more QoL than anything, if they end up getting more stuff in their vendor then its very easy to remove but as of now I'd just like it mining had more stuff to buy for how much work they do
- For the merc gear, if it does get replaced, I only changed about 3 lines of code for the vest, including its price in the vendor, so its a very easy fix for when the merc stuff gets replaced with salvage themed stuff, but as of now its very strong, before nerfing it you basically got an objectively better sec vest for 200 tickets, even now with its lower values its kinda just better than most of sec's vest options, I think the only way this would affect admemes is the lowered stam resist, its still a very good vest and very much still usable for admemes.
- I only slightly raised the Mining hardsuits, I do not intend to make them overpowered whatsoever, even with their upped stats, all this really does is help them not get turned into a fine red mist by hivelords and goliaths, these are not great for antags, their pierce resist is near nonexistant so sec can drop them with no issue, they have no stamina resist, 10% more resist so they dont get eaten alive isnt gonna suddenly make it an antag magnet in my opinion

Rinary1 reviewed (changes requested):
Why salvages should have syndie pistol in their vend?(Stechkin it's syndie thing)

R3v3l4t1on commented:
> Why salvages should have syndie pistol in their vend?(Stechkin it's syndie thing)

because they swapped the JHawk and the Stechkin completely. If you look into the PR you'll see stechkin is now sec/salv contra, and the JHawk is syndie contra. There should probably be some changes in uplink to reflect this but it's overall an attempt at nerfing salv to not have gear that is on par/better than sec's gear.

Rinary1 commented:
> > Why salvages should have syndie pistol in their vend?(Stechkin it's syndie thing)
> 
> because they swapped the JHawk and the Stechkin completely. If you look into the PR you'll see stechkin is now sec/salv contra, and the JHawk is syndie contra. There should probably be some changes in uplink to reflect this but it's overall an attempt at nerfing salv to not have gear that is on par/better than sec's gear.

Stechkin shouldn't be sec/salv, it's syndie only thing it's just a stupid thing that you'll changing one weapon to another, jhawk and stechkin isn't the same, stechkin has suppressor which also helps in stealth for syndie, when salvages/sec don't need it.

R3v3l4t1on commented on Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:
```diff
@@ -141,9 +141,9 @@
   - type: Armor
     modifiers:
       coefficients:
```
I'd wrap this in region starlight and end region tags. Looks cleaner.

R3v3l4t1on commented on Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:
```diff
@@ -213,50 +213,54 @@
     level: Metal
   - type: Armor
     modifiers:
       coefficients:
```
same here, wrap these values in a region instead of starlight edit behind them all.

R3v3l4t1on commented on Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:
```diff
@@ -213,50 +213,54 @@
+    sprintModifier: 0.85 #Starlight edit
   - type: ExplosionResistance
-    damageCoefficient: 0.2
+    damageCoefficient: 0.4 #Starlight edit
```
Here it's fine though.

R3v3l4t1on commented on Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:
```diff
@@ -213,50 +213,54 @@
-    useRate: 0
-  - type: UseDelay
-    delay: 10.0
+  #- type: ContainerContainer
```
add a region starlight around this commented out section so we know it was us that commented this out.

R3v3l4t1on commented on Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/mining.yml:
```diff
@@ -121,6 +121,16 @@
   conditions:
   - !type:BuyerAccessCondition
 
+- type: listing
```
Not a fan of this personally. I get it, these are real nice to have. But at least add a limit to how many you can buy. Science can research these after all.

R3v3l4t1on commented on Resources/Prototypes/_StarLight/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:
```diff
@@ -400,7 +400,7 @@
   parent: [BaseWeaponBattery, BaseGunWieldable, BaseSecuritySalvageContraband]
   id: WeaponLaserImmolator
-  description: Favoured by Nanotrasen Security for being cheap and easy to use.
+  description: An irregular, unstable firearm. It remains warm to the touch long after it has finished firing.
```
Bit of a random change in this PR but I like the new description.

R3v3l4t1on commented on Resources/Prototypes/_StarLight/Entities/Objects/Weapons/Guns/Pistols/pistols.yml:
I'll add context to this specifically:
This prototype of the Stechkin is a special 1-off variant for the Warden on Starboard. That being said: If we're making Stechkin a crew gun, this special prototype should be removed and the original one should become sec/salv contra. And perhaps it should also move away from the uplink? I'm unsure. I don't mind seeing the JHawk go for another sidearm but I am unsure if this is the appropriate way to move forward.

R3v3l4t1on replied:
On second thought, both protos should remain but one spawns with SP, which Salv buys. The other spawns with FMJ, and remains in ward's locker on starboard.

RorororoIDK commented:
> Why salvages should have syndie pistol in their vend?(Stechkin it's syndie thing)

funnily enough that stechkin was sec contra, i just added the salv tag and gave jhawk back its syndie tag

R3v3l4t1on commented on Resources/Prototypes/_StarLight/Entities/Objects/Weapons/Guns/Pistols/pistols.yml:
```diff
@@ -268,7 +269,7 @@
 - type: entity
   name: Jungle Hawk
-  parent: [ BaseSecuritySalvageContraband, WeaponPistolN1984 ]
+  parent: [ BaseSyndicateContraband, WeaponPistolN1984 ]
```
Maybe add to syndie uplink if not already there?

DisposalUnit commented:
> > > Why salvages should have syndie pistol in their vend?(Stechkin it's syndie thing)
> > 
> > 
> > because they swapped the JHawk and the Stechkin completely. If you look into the PR you'll see stechkin is now sec/salv contra, and the JHawk is syndie contra. There should probably be some changes in uplink to reflect this but it's overall an attempt at nerfing salv to not have gear that is on par/better than sec's gear.
> 
> Stechkin shouldn't be sec/salv, it's syndie only thing it's just a stupid thing that you'll changing one weapon to another, jhawk and stechkin isn't the same, stechkin has suppressor which also helps in stealth for syndie, when salvages/sec don't need it.

Okay so as much as controversial is this but
Warden on Starboard gets a syndicate weapon holster with a Stechkin inside of it.
The holster itself was renamed to be "warden's", as well as it's in-game description, however it still uses the same syndicate contra tag and acts as syndicate contra.

R3v3l4t1on reviewed (changes requested):
Some comments about how you're marking stuff as starlight, but mostly some concerns about your changes to a 1-off gun to make it mainstream. I don't think sec needs the headache of seeing salv walk around with Stechkin and wondering if it's the syndie version or the Sec/Salv one. But I'm honest to god unsure how to fix that, though I tried to make a suggestion.

RorororoIDK commented:
> > > Why salvages should have syndie pistol in their vend?(Stechkin it's syndie thing)
> > 
> > 
> > because they swapped the JHawk and the Stechkin completely. If you look into the PR you'll see stechkin is now sec/salv contra, and the JHawk is syndie contra. There should probably be some changes in uplink to reflect this but it's overall an attempt at nerfing salv to not have gear that is on par/better than sec's gear.
> 
> Stechkin shouldn't be sec/salv, it's syndie only thing it's just a stupid thing that you'll changing one weapon to another, jhawk and stechkin isn't the same, stechkin has suppressor which also helps in stealth for syndie, when salvages/sec don't need it.

i did look for literally any other pistol to put in there and it just boiled down to "either i give them the MK58 which is secs gun or the stechkin, because they cant keep the Jhawk and every revolver in the game has .45 magnum so i cant give em that either" the reason the Jhawk was added in the first place is because Avali and Resomi cant use the other sidearm in the vendor because it uses shotgun shells and i get that, being locked out of options isnt fun, i wasnt keen on giving them the stechkin either since im basically swapping uplink gun for uplink gun, but at least this one made more sense than the Jhawk, wasnt overpowered, and actually had some sort of link to commies which salv are so obsessed with

Rinary1 commented:
> > > > Why salvages should have syndie pistol in their vend?(Stechkin it's syndie thing)
> > > 
> > > 
> > > because they swapped the JHawk and the Stechkin completely. If you look into the PR you'll see stechkin is now sec/salv contra, and the JHawk is syndie contra. There should probably be some changes in uplink to reflect this but it's overall an attempt at nerfing salv to not have gear that is on par/better than sec's gear.
> > 
> > Stechkin shouldn't be sec/salv, it's syndie only thing it's just a stupid thing that you'll changing one weapon to another, jhawk and stechkin isn't the same, stechkin has suppressor which also helps in stealth for syndie, when salvages/sec don't need it.
> 
> Okay so as much as controversial is this but
> Warden on Starboard gets a syndicate weapon holster with a Stechkin inside of it.
> The holster itself was renamed to be "warden's", as well as it's in-game description, however it still uses the same syndicate contra tag and acts as syndicate contra. 

It's mappers + maints issue, when i added stechkin it was a syndie only weapon

Rinary1 commented:
> > > > Why salvages should have syndie pistol in their vend?(Stechkin it's syndie thing)
> > > 
> > > 
> > > because they swapped the JHawk and the Stechkin completely. If you look into the PR you'll see stechkin is now sec/salv contra, and the JHawk is syndie contra. There should probably be some changes in uplink to reflect this but it's overall an attempt at nerfing salv to not have gear that is on par/better than sec's gear.
> > 
> > Stechkin shouldn't be sec/salv, it's syndie only thing it's just a stupid thing that you'll changing one weapon to another, jhawk and stechkin isn't the same, stechkin has suppressor which also helps in stealth for syndie, when salvages/sec don't need it.
> 
> i did look for literally any other pistol to put in there and it just boiled down to "either i give them the MK58 which is secs gun or the stechkin, because they cant keep the Jhawk and every revolver in the game has .45 magnum so i cant give em that either" the reason the Jhawk was added in the first place is because Avali and Resomi cant use the other sidearm in the vendor because it uses shotgun shells and i get that, being locked out of options isnt fun, i wasnt keen on giving them the stechkin either since im basically swapping uplink gun for uplink gun, but at least this one made more sense than the Jhawk, wasnt overpowered, and actually had some sort of link to commies which salv are so obsessed with

Give mk58 instead of stechkin, that's all.

RorororoIDK commented:
> > > > > Why salvages should have syndie pistol in their vend?(Stechkin it's syndie thing)
> > > > 
> > > > 
> > > > because they swapped the JHawk and the Stechkin completely. If you look into the PR you'll see stechkin is now sec/salv contra, and the JHawk is syndie contra. There should probably be some changes in uplink to reflect this but it's overall an attempt at nerfing salv to not have gear that is on par/better than sec's gear.
> > > 
> > > 
> > > Stechkin shouldn't be sec/salv, it's syndie only thing it's just a stupid thing that you'll changing one weapon to another, jhawk and stechkin isn't the same, stechkin has suppressor which also helps in stealth for syndie, when salvages/sec don't need it.
> > 
> > 
> > i did look for literally any other pistol to put in there and it just boiled down to "either i give them the MK58 which is secs gun or the stechkin, because they cant keep the Jhawk and every revolver in the game has .45 magnum so i cant give em that either" the reason the Jhawk was added in the first place is because Avali and Resomi cant use the other sidearm in the vendor because it uses shotgun shells and i get that, being locked out of options isnt fun, i wasnt keen on giving them the stechkin either since im basically swapping uplink gun for uplink gun, but at least this one made more sense than the Jhawk, wasnt overpowered, and actually had some sort of link to commies which salv are so obsessed with
> 
> Give mk58 instead of stechkin, that's all.

im cool with that, will do

RorororoIDK committed:
Changes to ticket vendor and undone stechkin changes

removed advanced mineral scanner, swapped stechkin for MK58, swapped .40sp ammo for .35sp, reverted stechkin changes and changed the MK58 to be salvsec contra
